db/migrate/20181219021651_change_ticket.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeTicket < ActiveRecord::Migration[5.2]
+  def change
+    change_table :tickets do |t|
+      t.remove :stop_point_id
```

Comment:
why use this on the first pull of migration

Reply:
e sửa thành lưu tên bến luôn thì dễ hơn khi truy vấn và không cần thiết phải để quan hệ cho 2 bảng .